index.html

```diff
@@ -54,43 +59,46 @@ <h1>Canine Crusaders</h1>
                 </div>
                 <button id="search-button">Search</button>
             </div>
-                    
+
         </aside>
-<!-- Adopt a Pet container with dog adoption API showcase-->
```

Comment:
@dachyon1123 Will you please review this part. I have the code that is in red that will be deleted upon merge. Please make sure this is correct.